nekoyume/Assets/StreamingAssets/Localization/character_name.csv

```diff
@@ -26,7 +26,7 @@ CHARACTER_NAME_203003,Pig Tribe Warrior,돼지부족 전사,Porco Guerreiro,,ピ
 CHARACTER_NAME_203004,Pig Tribe Paladin,돼지부족 성기사,Porco Paladino,,ピッグ・パラディン,Schweinepaladin,Paladín cerdo,หมูพาลาดิน,Świński paladyn,,Varken Paladijn,Babi Paladin,Gardien Cochon,Паладин Пиг,ជ្រូកផាឡាដិន,,猪圣骑士,Disznó lovag
 CHARACTER_NAME_203005,Pig Tribe Butcher,돼지부족 도살자,Porco Invasor,,ピッグ・ブッチャー,Schweineschlachter,Intruso cerdo,หมูเนื้อ,Świński bandyta,,Varken Slager,Babi Penjajah,Boucher Cochon,Мясник Пиг,ជ្រូកឃាតករ,,猪闯入者,Disznó mészáros
 CHARACTER_NAME_203006,Pig Tribe Wizard,돼지부족 마도사,Porco Gatuno,,ピッグ・ウィザード,Schweinedieb,Ladrón cerdo,จอมโจรหมู,Świński złodziej,,Varken Tovenaar,Babi Penjagal,Sorcier Cochon,Волшебник Пиг,ជ្រូកលួច,,猪巫师,Disznó varázsló
-CHARACTER_NAME_203007,Saehrimnir,제림니르,Saehrimnir,,セリムニル,Saehrimnir,Saehrimnir,Saehrimnir,Saehrimnir,,Saehrimnir,Saehrimnir,Saehrimnir,Серимнир,Saehrimnir,,塞里米尔,Saehrimnir
+CHARACTER_NAME_203007,Saehrimnir,제림니르,Sæhrímnir,,セーフリームニル,Sæhrímnir,Sæhrimnir,Saehrimnir,Sahrimnir,Saehrimnir,Saehrimnir,Saehrimnir,Sæhrímnir,сехримнир,Saehrimnir,,沙赫利姆尼尔,
```

Comment:
@Oremal Some font can't print 'æ', so we have to write this 'ae'.

Reply:
done